src/charm.py

```diff
@@ -327,12 +327,14 @@ def _prometheus_config(self) -> str:
             "scrape_timeout": "5s",
             "metrics_path": "/metrics",
             "honor_timestamps": True,
+            "honor_labels": True,
```

Comment:
This is not needed. The default self-monitoring job comes with no labels we override in its scraping.

Reply:
I will confess I knew that. I was being lazy in writing the test case. By adding this line here I could make my test case simple. Without this line I would need extra code in my test case to exclude the self-monitoring job.

Ping here, if you think there is cause for concern here that I have not yet comprehended and I will stop being lazy.

Reply:
Yes please, fix it. 👍🏽

Reply:
Ok. Will refactor test to only check for honor labels key being set only in non-self monitoring jobs.